go.mod

```diff
@@ -3,26 +3,21 @@ module github.com/keikoproj/active-monitor
 go 1.12
```

Comment:
.travis.yaml has go 1.13.x and go mod has go 1.12. can we go.mod also at 1.13?

Reply:
Done